src/lib/tuning/index/column_ref.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * A ColumnRef is used to reference a set of columns by a table name
+ * and their column indices in this table
+ */
+struct ColumnRef {
```

Comment:
Could you please provide more motivation and background of the necessity of this class?

Reply:
This is used by the IndexEvaluator, IndexChoices and IndexOperations. It may look very similar to an `IndexInfo`, but this does not reference the table together with its relevant columns. Would it make sense to combine both (add the `table_name` to `IndexInfo`)?